src/components/SinglePastCommitInfo.tsx

```diff
@@ -217,15 +217,16 @@ export class SinglePastCommitInfo extends React.Component<
                     <button
                       className={`${diffIconStyle}`}
                       title={'View file changes'}
-                      onClick={() => {
-                        openDiffView(
+                      onClick={async () => {
+                        await openDiffView(
```

Comment:
This was needed because `openDiffView` is now async
https://github.com/jupyterlab/jupyterlab-git/pull/395/files#diff-114241b8de041aad5f23f605270e246bR22